ajax/libs/sanctuary/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "sanctuary",
+  "description": "Refuge from unsafe JavaScript",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sanctuary-js/sanctuary.git"
+  },
+  "dependencies": {
+    "sanctuary-def": "0.20.1",
+    "sanctuary-either": "1.2.0",
+    "sanctuary-maybe": "1.2.0",
+    "sanctuary-pair": "1.2.0",
+    "sanctuary-show": "1.0.0",
+    "sanctuary-type-classes": "11.0.0",
+    "sanctuary-type-identifiers": "2.0.1"
+  },
+  "devDependencies": {
+    "fantasy-land": "4.0.1",
+    "jsverify": "0.8.x",
+    "sanctuary-descending": "1.2.0",
+    "sanctuary-identity": "1.2.x",
+    "sanctuary-scripts": "4.0.x"
+  },
```

Comment:
Can `devDependencies` safely be omitted?

Reply:
Yup.

Reply:
Thanks. I have removed `devDependencies`. :)